hazelcast/src/main/java/com/hazelcast/internal/diagnostics/OperationProfilerPlugin.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.diagnostics;
+
+
+import com.hazelcast.internal.util.LatencyDistribution;
+import com.hazelcast.spi.impl.NodeEngineImpl;
+import com.hazelcast.spi.impl.operationservice.impl.OperationServiceImpl;
+import com.hazelcast.spi.properties.HazelcastProperty;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentMap;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+/**
+ * A {@link DiagnosticsPlugin} that displays operation latency information.
+ */
+public class OperationProfilerPlugin extends DiagnosticsPlugin {
```

Comment:
Minor: this class looks almost exactly the same as `InvocationProfilerPlugin`, I guess it can be easily deduplicated.

Reply:
Fixed